uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/DAO/ContactWorkPattern.php

```diff
@@ -173,16 +173,16 @@ static function &fields() {
           'name' => 'change_reason',
           'type' => CRM_Utils_Type::T_STRING,
           'title' => ts('Change Reason') ,
-          'description' => 'One of the values of the Job Contract Revision Change Reason option group',
+          'description' => 'One of the values of the Work Pattern Change Reasons option group',
           'maxlength' => 512,
           'size' => CRM_Utils_Type::HUGE,
           'table_name' => 'civicrm_hrleaveandabsences_contact_work_pattern',
           'entity' => 'ContactWorkPattern',
           'bao' => 'CRM_HRLeaveAndAbsences_DAO_ContactWorkPattern',
           'localizable' => 0,
           'pseudoconstant' => array(
-            'optionGroupName' => 'hrjc_revision_change_reason',
-            'optionEditPath' => 'civicrm/admin/options/hrjc_revision_change_reason',
+            'optionGroupName' => 'hrleaveandabsences_work_pattern_change_reason',
+            'optionEditPath' => 'civicrm/admin/options/hrleaveandabsences_work_pattern_change_reason',
           )
```

Comment:
How did you update this file? It has to be updated using the script in bin/build-dao.sh. Is this how you did it?

Reply:
No. I updated manually. I though i can do it from here.